services/minespace-web/src/components/dashboard/mine/reports/Reports.tsx

```diff
@@ -0,0 +1,202 @@
+import React, { FC, useEffect, useState } from "react";
+import { connect } from "react-redux";
+import { bindActionCreators } from "redux";
+import PlusCircleFilled from "@ant-design/icons/PlusCircleFilled";
+import { Button, Col, Row, Typography } from "antd";
+import moment from "moment";
+import PropTypes from "prop-types";
+import {
+  createMineReport,
+  fetchMineReports,
+  updateMineReport,
+} from "@mds/common/redux/actionCreators/reportActionCreator";
+import { closeModal, openModal } from "@mds/common/redux/actions/modalActions";
+import { getMineReports } from "@mds/common/redux/selectors/reportSelectors";
+import { getMineReportDefinitionOptions } from "@mds/common/redux/reducers/staticContentReducer";
+import CustomPropTypes from "@/customPropTypes";
+import ReportsTable from "@/components/dashboard/mine/reports/ReportsTable";
+import { modalConfig } from "@/components/modalContent/config";
+import AuthorizationWrapper from "@/components/common/wrappers/AuthorizationWrapper";
+import { IMine, IMineReport } from "@mds/common";
+import { ActionCreator } from "@mds/common/interfaces/actionCreator";
+
+const propTypes = {
+  mine: CustomPropTypes.mine.isRequired,
+  mineReports: PropTypes.arrayOf(CustomPropTypes.mineReport).isRequired, // eslint-disable-next-line react/no-unused-prop-types
+  mineReportDefinitionOptions: PropTypes.arrayOf(PropTypes.objectOf(PropTypes.any)).isRequired,
+  updateMineReport: PropTypes.func.isRequired,
+  createMineReport: PropTypes.func.isRequired,
+  fetchMineReports: PropTypes.func.isRequired,
+  openModal: PropTypes.func.isRequired,
+  closeModal: PropTypes.func.isRequired,
+};
```

Comment:
Are these propTypes still required since there is already an interface with the same attributes defined below?